test/Microsoft.ML.CpuMath.PerformanceTests/Microsoft.ML.CpuMath.PerformanceTests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.2</LangVersion>
+    <StartupObject>Microsoft.ML.CpuMath.PerformanceTests.Program</StartupObject>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Let's chat about this tomorrow.  This isn't going to work in CI, nor when normal developers try to build.  The reason is because they won't have the 3.0 SDK installed.  We will basically need to skip this project and the UnitTests.netcoreapp project.